lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,74 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
   margin: 0;
-  padding-top: 20px;
   position: relative; // to make a containing block for validation popovers
 }
 
-.nx-form--simple {
-  border-top: none;
-  padding: 0;
-}
+.nx-form-footer {
+  @include container-horizontal;
 
-// for a form that is the entirety of a modal
-.nx-form--modal-form {
-  padding-top: 0;
+  border-top: 1px solid #e9e9e9;
+  padding: $nx-spacing-xl $nx-spacing-l;
+  text-align: right;
 }
 
 .nx-form-group {
-  margin-bottom: 20px;
+  margin-bottom: 6px;
 }
 
 .nx-form-row {
   @include container-horizontal;
-  margin-bottom: 24px;
+  margin-bottom: 6px;
 
   .nx-form-group, .nx-fieldset {
     display: inline-block;
-    margin-right: 24px;
-    margin-bottom: 0;
-    vertical-align: bottom; // fixes issue where certain form types display lower
-
-    &:last-child {
-      margin-right: 0;
-    }
-  }
-
-  .nx-label {
+    margin-right: $nx-spacing-l;
     margin-bottom: 0;
   }
 }
 
-.nx-btn-bar--forms {
-  border-top: $nx-border-dark;
-  padding-top: 8px;
-}
-
 // labels
 
 .nx-label, .nx-legend {
   @include semi-bold();
 
   color: $nx-text-color;
-  display: block;
-  font-size: $nx-font-size;
-  margin-bottom: $nx-spacing-xs;
+  padding: 0;
```

Comment:
Seems redundant

Reply:
> Seems redundant

The browser agent adds 2px of padding to `<legend>`, I just removed it here for both for clarity.